src/services/chatsService.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const { Op } = require('sequelize');
+const { Chat } = require('../models/Chat');
+const { Message } = require('../models/Message');
+
+function normalize({ id, name, creatorName, members }) {
+  return {
+    id, name, creatorName, members,
+  };
+}
+
+async function getById(id) {
+  const foundChat = await Chat.findByPk(id);
+
+  return foundChat;
+}
+
+async function getByUser(userName) {
+  const foundMessages = await Chat.findAll({
+    where: {
+      members: {
+        [Op.contains]: [userName],
+      },
+    },
+    order: [['updatedAt', 'DESC']],
+    attributes: { exclude: ['updatedAt', 'createdAt'] }, // normalized
```

Comment:
```suggestion
    attributes: { exclude: ['updatedAt', 'createdAt'] }, 
```
Remove all the comments!